fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1626,6 +1629,7 @@ public void loadImage() throws IOException {
                 .put(SRMetaBlockID.WAREHOUSE_MGR, warehouseMgr::load)
                 .put(SRMetaBlockID.CLUSTER_SNAPSHOT_MGR, clusterSnapshotMgr::load)
                 .put(SRMetaBlockID.BLACKLIST_MGR, sqlBlackList::load)
+                .put(SRMetaBlockID.DIGEST_BLACKLIST_MGR, sqlDigestBlackList::load)
```

Comment:
The order of `sqlDigestBlackList` is inconsistent between `saveImage()` and `loadImage()`